packages/safe-ds-vscode/src/extension/messages.ts

```diff
@@ -68,7 +68,17 @@ export interface ProgramMainInformation {
 export interface PlaceholderQueryMessage {
     type: 'placeholder_query';
     id: string;
-    data: string;
+    data: PlaceholderQuery;
+}
+
+/**
+ * Contains the query for a placeholder value.
+ * Field name contains the name of the requested placeholder, field window_begin may contain an offset of and window_size may contain the size of a subset of the requested data.
```

Comment:
Better add a `/** ... */` comment to the individual fields.

Reply:
Where applicable, comments for other messages were also restructured.